src/keys/bip39.rs

```diff
@@ -247,12 +277,24 @@ impl fmt::Debug for Passphrase {
 #[derive(Clone, Zeroize, ZeroizeOnDrop)]
 pub struct Seed([u8; 64]);
 
+impl Seed {
+    pub fn as_bytes(&self) -> &[u8; 64] {
+        &self.0
+    }
+}
+
 impl AsRef<[u8; 64]> for Seed {
     fn as_ref(&self) -> &[u8; 64] {
         &self.0
     }
 }
 
+impl AsRef<[u8]> for Seed {
```

Comment:
I would probably stick to the above impl. Having both will make this unnecessarily difficult to use.

Reply:
Right. I thought to make seed generic in `Slip10::from_seed<S: AsRef<[u8]>>(seed: &S)`. I'll remove `AsRef<[u8; 64]>` impl then - there's `fn to_bytes(&self) -> &[u8; 64]` as a replacement.